lib/util/crypto.js

```diff
@@ -38,10 +38,8 @@ const verifyPassword = (plain, hash) => ((isBlank(plain) || (isBlank(hash)))
   ? resolve(false)
   : bcrypt.compare(plain, hash));
 
-// Returns a cryptographically random base64 string of a given length.
-const generateToken = (bytes = 48) => randomBytes(bytes).toString('base64')
-  .replace(/\//g, '!')
-  .replace(/\+/g, '$');
+// Returns a cryptographically random base64url string of a given length.
+const generateToken = (bytes = 48) => randomBytes(bytes).toString('base64url');
```

Comment:
In addition to the new comment in lib/http/preprocessors.js, I think it'd be useful to add a comment here saying that this function used to return tokens that contained `$` and `!`. That way, someone looking at this function to try to understand Central tokens would understand all the possible characters that a token in use could contain.